### ISSUE TITLE
Unable To View Afflicted Affix Bars

### ISSUE THREAD

**Issue by martheli** [Backupiseasy/ThreatPlates#469]
I turned off Threatplates and the health bars showed up so theres something with threat plates thats hiding them. I have these settings:

![image](https://github.com/Backupiseasy/ThreatPlates/assets/20176169/04ed6b74-d417-487c-8621-e247b1807b92)

I tried turning off show blizzard nameplates for friendly units and it still wasn't showing up.

**Comment by Backupiseasy** [comment]
I think you need to enable minion nameplates for that to work. There is currently no option for that in Threat Plates - not sure why, but that's how it is. You can do that in the options of WoW (search for Minions in nameplates) or by using this command:

`/script SetCVar("nameplateShowFriendlyMinions", 1)`

**Comment by martheli** [comment]
I have friendly minion nameplates turned on in WoW interface options already.

![image](https://github.com/Backupiseasy/ThreatPlates/assets/20176169/f6b7c976-ce1a-4687-a3c5-21a435f78dbc)

**Comment by Backupiseasy** [comment]
Hm, then I need to investigate some more, I guess.

**Comment by martheli** [comment]
any additional follow up on this?

**Comment by Backupiseasy** [comment]
No, not yet. I had to fix some bugs in the current version and did not have time to look into this. 

You said you see healthbars when you disable Threat Plates, right? As you have enabled minion nameplates already it should be like that. In that case, you are right, there should be something in Threat Plates that hides them.

**Comment by martheli** [comment]
is anyone able to view afflicted mobs using threat plates? Is this something i'm doing wrong or does threat plates not have a way to show/hide these mobs nameplates?
